### PR TITLE
Ref #17

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
Update spack.

### CODE REVIEW

File: ci/spack.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ spack:
   - netcdf-fortran@4.5.3
   - w3emc@2.9.2
   - gsi-ncdiag@1.0.0
+  - bacio@2.4.1
```
Comment (aerorahul) [inline ci/spack.yaml]: I think you want to move this above the `w3emc` line.  I don't think it matters, but since it is a dependency, it makes sense.

**Review by aerorahul — APPROVED** [review]
Technically, everytime you update this file, you should flush the `cache`. 
Since this dependency was already satisfied, and there is no change in versions in the CI, you don't have to flush the cache.

Refer to the notes in the workflow yaml files on how to flush the CI cache.

File: ci/spack.yaml (hunk 1 above)
Comment (EdwardSafford-NOAA) [reply ci/spack.yaml]: Ok, I'll make that change.  Thanks.

**Comment by EdwardSafford-NOAA** [comment]
Flushed cache, per instructions and learned how to set my PAT scope to include "workflow" in allow a commit.

**Pull request merged**